### PR TITLE
Update UI for windows

### PR DESCRIPTION
**Pull request by k-tten**
Mainly added more space between form inputs since the animation when an input required space above the input

Moved settings icon to the bottom left, removed text

Flexbox for making the version and type selection more fluid

Fixed typos on file names

### CODE REVIEW

**Comment by ariscript** [comment]
There is a small but *very annoying* scrollbar in the new instance window:
image

Other than that, it looks good to merge!

File: src/windows/NewInstanceWindow.tsx [hunk 1]
```diff
@@ -80,10 +78,10 @@ const NewInstanceWindow = () => {
                     {err}
                 </div>
             )}
-            <h2 className="font-xl font-bold mb-2">New Instance</h2>
+            <h2 className="font-2xl font-bold mb-2">New Instance</h2>
             <form
                 onSubmit={handleFormSubmit}
-                className="flex container flex-col space-y-1"
+                className="flex container flex-col gap-4"
```
Comment (ariscript) [inline src/windows/NewInstanceWindow.tsx]: This gap is probably too large and creates a scrollbar on the window.
```suggestion
                className="flex container flex-col gap-3"
```

**Review by ariscript — CHANGES_REQUESTED** [review]
Other than that one issue, this PR is good to go

**Comment by k-tten** [comment]
Ok it looks like it worked (on my machine). I have committed the new suggestion.

**Review by ariscript — APPROVED** [review]
Looks good!

**Pull request merged**